### ISSUE TITLE
UptimeRobot

### ISSUE THREAD

**Issue by abjerner** [leekelleher/umbraco-package-ideas#4]
Should be possible to make something useful here. Current progress for tonight:

![image](https://cloud.githubusercontent.com/assets/3634580/11546299/b835d42a-994c-11e5-86fb-a2db08a56ffe.png)

Feel free to assign me the issue, and I'll look further into it ;)

**Comment by greystate** [comment]
Haha - totally my new favorite site, that one, right there :-)

**Comment by abjerner** [comment]
Same here. Hadn't heard about it before, but it allows more monitors at the free plan and the paid service also allows more monitors than our current service, and there is an API :D

**Comment by leekelleher** [comment]
http://omgbacon.dk/ Haha! Love it.

I use UptimeRobot for a lot of my client sites.

I'll update the ideas page in the morning. Thanks @abjerner!

**Comment by greystate** [comment]
LOL - I was referring to omgbacon.dk ! ! !

**Comment by abjerner** [comment]
Ahh, I see Chriztian. That one I already knew about ;)

**Comment by leekelleher** [comment]
[I've updated the dashboards list](https://github.com/leekelleher/umbraco-package-ideas/blob/master/ideas/backoffice-dashboards.md).

**Comment by abjerner** [comment]
I forgot to add my progress here, so here goes:

I did some work on it back in December, and have released some of this as an [UptimeRobot API implementation](https://github.com/abjerner/Skybrud.Social.UptimeRobot) package. This is a standalone package in the way that it doesn't require an Umbraco installation, since it i just the "service layer" if you will. I'm using it internally for monitoring some of our solutions, and haven't had any issues so far.

For the Umbraco dashboard screenshot I posted earlier, the plan was to make this a part of [Umbraco Dashboard package](https://github.com/abjerner/Skybrud.Umbraco.Dashboard). I haven't worked on it since before Christmas, but I will try to find some time to wrap up the code and push it to the repository ;)
